profiling/src/profile/profiled_endpoints.rs

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
+
+use std::collections::HashMap;
+
+use serde::ser::SerializeMap;
+use serde::Serialize;
+
+#[derive(Default, PartialEq, Eq, Debug, Clone)]
+pub struct ProfiledEndpointsStats {
+    count: HashMap<String, i64>,
+}
+
+impl Serialize for ProfiledEndpointsStats {
```

Comment:
I think you could use `#[serde(transparent)]` https://serde.rs/container-attrs.html#transparent  on the struct instead of implementing serialization

Reply:
I may miss something, I get this error
![image](https://user-images.githubusercontent.com/17292193/203097731-8e702140-ed94-41ea-98ec-fd13fa6db5b2.png)
and cannot find a way to fix it. Do you have any idea?

Reply:
The struct needs to be `#[derive(Serde)] and `#[serde(transparent)]` is a struct level attribute.

This works for me.

```
#[derive(Default, PartialEq, Eq, Debug, Clone, Serialize)]
#[serde(transparent)]
pub struct ProfiledEndpointsStats {
    count: HashMap<String, i64>,
}
```

Reply:
✔️